projects/ng-core-tester/src/app/record/editor/schema.json

```
@@ -0,0 +1,396 @@
+            "value": "value 2",
+            "label": "val2"
+          },
+          {
+            "value": "value 3",
+            "label": "val3"
+          }
+        ]
+      }
+    },
+    "object": {
+      "title": "Object",
+      "type": "object",
+      "additionalProperties": false,
+      "propertiesOrder": [
+        "prop1",
+        "prop2"
+      ],
+      "required": [
+        "prop1"
+      ],
+      "properties": {
+        "prop1": {
+          "title": "Property 1",
+          "type": "string"
+        },
+        "prop2": {
+          "title": "Property 2",
+          "type": "string"
+        }
+      },
+      "form": {
+        "templateOptions": {
+          "cssClass": "editor-title"
+        }
+      }
+    },
+    "object_of_object": {
+      "title": "Object of Object",
+      "type": "object",
+      "additionalProperties": false,
+      "propertiesOrder": [
+        "object1",
+        "prop3"
+      ],
+      "properties": {
+        "object1": {
+          "title": "Object1",
+          "type": "object",
+          "additionalProperties": false,
+          "propertiesOrder": [
+            "prop1",
+            "prop2"
+          ],
+          "required": [
+            "prop1"
+          ],
+          "properties": {
+            "prop1": {
+              "title": "Property 1",
+              "type": "string"
+            },
+            "prop2": {
+              "title": "Property 2",
+              "type": "string"
+            }
+          }
+        },
+        "prop3": {
+          "title": "Property 3",
+          "type": "string"
+        }
+      },
+      "form": {
+        "templateOptions": {
+          "cssClass": "editor-title"
+        }
+      }
+    },
+    "array": {
+      "title": "Array",
+      "type": "array",
+      "minItems": 1,
+      "items": {
+        "title": "properties",
+        "type": "object",
+        "additionalProperties": false,
+        "propertiesOrder": [
+          "prop1",
+          "prop2"
+        ],
+        "required": [
+          "prop1"
+        ],
+        "properties": {
+          "prop1": {
+            "title": "Prop 1",
+            "type": "string",
+            "form": {
+              "templateOptions": {
+                "cssClass": "col"
+              }
+            }
+          },
+          "prop2": {
+            "title": "Prop 2",
+            "type": "string",
+            "form": {
+              "hide": true,
+              "templateOptions": {
+                "cssClass": "col-lg-6"
+              }
+            }
+          }
+        },
+        "form": {
+          "templateOptions": {
+            "cssClass": "row"
+          }
+        }
+      },
+      "form": {
+        "templateOptions": {
+          "cssClass": "editor-title"
+        }
+      }
+    },
+    "array_of_array": {
+      "title": "Array Of Array",
+      "default": [
+        [
+          "value 1",
+          "value 2"
+        ],
+        [
+          "value 3"
```

I'm not sure here, should it have uppercases?
```suggestion
          "Value 1",
          "Value 2"
        ],
        [
          "Value 3"
```

---

It is up to us. They are values.